modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -89,8 +101,9 @@ The certificate must be set with a passphrase and a common name (CN) that specif
 
 ... Specify a value for *Key File*. This is the PEM formatted file that contains the private key for the certificate.
 ... If the *Key Passcode* field is displayed after uploading a PEM key file, enter the word or phrase that protects the private key.
-... Optionally, specify a value for *CA Path Certificate File (Optional)*. The CA path contains the intermediary and root certificates that provide the path (chain of trust) from the certificate to the root. When you provide an entry in this field, the path is sent along with the certificate during the TLS handshake.
-... Select *Deploy*. The *Key Passcode* field is blanked out for security reasons. You can still review public certificate details. If you upload a new key file, this field is again enabled.
+... Specify a value for *CA Path Certificate File (Optional)*. The CA path contains the intermediary and root certificates that provide the path (chain of trust) from the certificate to the root. The file is downloadable from the CA that signed your certificate. When you provide an entry in this field, the Runtime Fabric load balancer at every connection sends both the certificate and the path to the client. Many clients require the server to send the CA path so that the certificate can be validated.
+... Review the certificate details to verify domains, expiration, and the other configuration settings. By default, the first domain listed is the domain that is used when deploying applications.
```

Comment:
>By default, the first domain listed is the domain that is used when deploying applications.

This is true if only one certificate is added. I think it would be clearer to leave this detail out of this section.

Reply:
Chris was testing this and thought the first one listed was the default. (Perhaps I'm misremembering.)

Reply:
It is, but the context for this sentence is when you're uploading a certificate in a UI where multiple certificates can be uploaded. The first domain in the list of all domains across all uploaded certificates will be the default. But any one certificate they upload may not affect the default domain.

Reply:
Ah - OK - thank you for clarifying this! I'll remove it.